assets/js/main.js

```diff
@@ -49,6 +49,9 @@ function generateOnThisPage() {
         // It's hidden by default. If we added links to the list, show it.
         if (found) {
             $(".on-this-page").show();
+            $(".on-this-page").click(function(){
+                $(window).scrollTop(0); 
+            });
```

Comment:
You can literally delete this:

```suggestion
```

.. and make [this line](https://github.com/pulumi/docs/blob/master/layouts/partials/docs/right-nav.html#L4) be this instead:

```
<h4 class="no-anchor">
    <a href="#">On This Page</a>
</h4>
```

There's no need for any script here, and [semantically](https://developer.mozilla.org/en-US/docs/Glossary/Semantics), it should be an `a` tag with an `href` as well.

Reply:
>This is almost surely happening because PostCSS is removing the styles you've defined here, as they aren't being referenced in the markup. (We do this intentionally to keep our CSS from becoming bloated -- I believe this has been mentioned before.) 

Yes, it was mentioned before. For whatever reason, I had interpreted that as it having something to do specifically with the way we were generating the preview and that you were just calling it out as to why the preview wasn't working. I didn't realize something needed to be done to like whitelist the styles. I will go ahead and do that.

Reply:
>For whatever reason, I had interpreted that as it having something to do specifically with the way we were generating the preview

The preview is a prod build, so what you see in preview is what you'll see in prod, yeah. We should treat these as what the world will look like post-merge.

Reply:
BTW, you can get the same behavior by running `make build`, `cd`ing into `public`, and starting a local HTTP server. So you don't ave to actually push a commit to GitHub to see what'd happen in a preview.